repose-aggregator/components/services/datastore-service/datastore-service-impl/src/main/resources/META-INF/schema/config/dist-datastore-configuration.xsd

```diff
@@ -46,6 +47,73 @@
             <xs:element name="allowed-hosts" type="dds:HostAccessControlList" minOccurs="1" maxOccurs="1"/>
             <xs:element name="port-config" type="dds:PortConfiguration" minOccurs="1" maxOccurs="1"/>
         </xs:sequence>
+
+        <xs:attribute name="connection-pool-id" type="xs:string" use="optional">
+            <xs:annotation>
+                <xs:documentation>
+                    <html:p>
+                        HTTP Connection pool (ID) to use when communicating with other members of the distributed
+                        datastore.
+                    </html:p>
+                </xs:documentation>
+            </xs:annotation>
+        </xs:attribute>
+
+        <xs:attribute name="keystore-filename" type="xs:anyURI" use="optional">
+            <xs:annotation>
+                <xs:documentation>
+                    <html:p>
+                        IF this attribute is configured,
+                        THEN it is assumed that it points to the Java keystore containing the client certificate to
+                        present for client authentication (e.g keystore.jks)
+                        AND the keystore-password and key-password attributes are no longer optional.
+                    </html:p>
+                </xs:documentation>
+            </xs:annotation>
+        </xs:attribute>
+
+        <xs:attribute name="keystore-password" type="xs:string" use="optional">
+            <xs:annotation>
+                <xs:documentation>
+                    <html:p>The password for the client authentication keystore.</html:p>
+                </xs:documentation>
+            </xs:annotation>
+        </xs:attribute>
+
+        <xs:attribute name="key-password" type="xs:string" use="optional">
```

Comment:
This was actually mostly copy-paste configuration. I believe the key-password is optional in both the container configuration and the http connection pool configuration.